Makefile

```diff
@@ -0,0 +1,21 @@
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: dependencies
+dependencies: ## Install dependencies
+	pip install --user -r requirements.txt
+
```

Comment:
Don't we need `bundle install`?

Reply:
added bundle install, removed the --user